configure.ac

```diff
@@ -200,6 +201,7 @@ AX_PATH(domserver_logdir,       [$domserver_root/log])
 AX_PATH(domserver_rundir,       [$domserver_root/run])
 AX_PATH(domserver_tmpdir,       [$domserver_root/tmp])
 AX_PATH(domserver_exampleprobdir, [$domserver_root/example_problems])
+AX_PATH(domserver_databasedumpdir, [$domserver_root/db-dumps])
```

Comment:
Should this live somewhere under output by default?

Reply:
Isn't the folder now at the same level as the log? for the case that we install in `/opt` would you want it next to `/opt/domjudge/log`?

Reply:
`output` exists in maintainer-mode; I think this is fine in non-FHS installation mode.